lib/forge-std/.github/workflows/tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+  check:
+    name: Foundry project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
+        with:
+          version: nightly
+
+      - name: Install dependencies
+        run: forge install
+      - name: Run tests
+        run: forge test -vvv
+      - name: Build Test with older solc versions
+        run: |
+          forge build --contracts src/Test.sol --use solc:0.8.0
```

Comment:
Why do we do this? I don't think we really need to support different solidity versions?

Reply:
Should be clearer now that foundry is a submodule instead of hardcoded into the repo 🤦